### ISSUE TITLE
Asserts limit do not detect "Sinon" camelcase

### ISSUE THREAD

**Issue by ymoreiratiti** [onechiporenko/eslint-plugin-mocha-cleanup#45]
Somente tests where Sinon is imported with the name "Sinon" instead "sinon" returns `Test without assertions is not allowed.`

e.g.

```typescript
const rule = require('../../../lib/rules/asserts-limit');
const { RuleTester } = require('eslint');

const ruleTester = new RuleTester({ env: { es6: true } });

ruleTester.run('asserts-limit', rule, {
  valid: [
    {
      only: true,
      code: `it('1234', function () {
        Sinon.assert.calledOn(sp, {});
      })`
    }
  ],
  invalid: [],
});
```

It appears that changing [lib/utils/node.js#L36](https://github.com/onechiporenko/eslint-plugin-mocha-cleanup/blob/7f135e3ed8c3e353ea5eeadbf3cb21059ad87ce8/lib/utils/node.js#L36)
from `node.object.name === 'sinon'` to `String(node.object.name).toLowerCase() === 'sinon'` solves.
